web/packages/teleport/src/Roles/RoleEditor/StandardEditor/Resources.tsx

```diff
@@ -379,6 +382,60 @@ export function KubernetesAccessSection({
   );
 }
 
+function KubernetesResourceKindView({
+  value,
+  validation,
+  isProcessing,
+  onChange,
+  roleVersion,
+}: {
+  value: KubernetesResourceModel;
+  validation: KubernetesResourceValidationResult['kind'];
+  isProcessing: boolean;
+  onChange?(m: KubernetesResourceModel): void;
+  roleVersion: RoleVersion;
+}) {
+  const supportsCRDs = supportsKubernetesCustomResources(roleVersion);
```

Comment:
Nit: in our web code we only capitalize the first letter of an acronym.
```suggestion
  const supportsCrds = supportsKubernetesCustomResources(roleVersion);
```